protected/humhub/modules/space/views/space/about.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+use humhub\modules\space\models\Space;
+use humhub\modules\space\widgets\Members;
+use humhub\modules\space\widgets\MyMembership;
+use humhub\modules\space\widgets\CustomSidebar;
+use humhub\modules\content\widgets\richtext\RichText;
+use humhub\modules\user\widgets\Image;
+
+?>
+
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <?= Yii::t('SpaceModule.base', '<strong>About Space</strong>') ?>
+    </div>
+    <div class="panel-body">
+        <div>
```

Comment:
We should omit the description part completely if neither description is available.

Reply:
Just an idea, instead of `About Space` why not use the display name of the Space itself? After all you're wanting users to know it's about the given space :thinking: